### PR TITLE
Config Additions and Changes

### PR DESCRIPTION
**Pull request by Antinoe**
-Balanced some of the options.
-Fixed Client Config's title.
-Added options for Power Beam V1, V2, V3 and most Missile Launcher Addons.

### CODE REVIEW

File: Content/Items/Addons/V3/NebulaBeamAddon.cs [hunk 1]
```diff
@@ -36,9 +36,9 @@ public override void SetDefaults()
 			Item.createTile = ModContent.TileType<Content.Tiles.ItemTile.Beam.NebulaBeamTile>();
 			MGlobalItem mItem = Item.GetGlobalItem<MGlobalItem>();
 			mItem.addonSlotType = 2;
-			mItem.addonDmg = 2.25f;
-			mItem.addonHeat = 1f;
-			mItem.addonSpeed = 0;
+			mItem.addonDmg = Common.Configs.MConfigItems.Instance.damageIceBeam;
+			mItem.addonHeat = Common.Configs.MConfigItems.Instance.overheatIceBeam;
+			mItem.addonSpeed = Common.Configs.MConfigItems.Instance.speedIceBeam;
```
Comment (Armipotent) [inline Content/Items/Addons/V3/NebulaBeamAddon.cs]: Is the Nebula Beam pulling from Ice Beam values intentional?

File: Content/Items/Addons/V3/SolarBeamAddon.cs [hunk 2]
```diff
@@ -35,9 +35,9 @@ public override void SetDefaults()
 			Item.createTile = ModContent.TileType<Content.Tiles.ItemTile.Beam.SolarBeamTile>();
 			MGlobalItem mItem = Item.GetGlobalItem<MGlobalItem>();
 			mItem.addonSlotType = 4;
-			mItem.addonDmg = 3f;
-			mItem.addonHeat = 1.5f;
-			mItem.addonSpeed = -0.15f;
+			mItem.addonDmg = Common.Configs.MConfigItems.Instance.damageIceBeam;
+			mItem.addonHeat = Common.Configs.MConfigItems.Instance.overheatIceBeam;
+			mItem.addonSpeed = Common.Configs.MConfigItems.Instance.speedIceBeam;
```
Comment (Armipotent) [inline Content/Items/Addons/V3/SolarBeamAddon.cs]: Is the Solar Beam pulling from Ice Beam values intentional as well?

File: Content/Items/Addons/V3/StardustBeamAddon.cs [hunk 3]
```diff
@@ -36,9 +36,9 @@ public override void SetDefaults()
 			Item.createTile = ModContent.TileType<Content.Tiles.ItemTile.Beam.StardustBeamTile>();
 			MGlobalItem mItem = Item.GetGlobalItem<MGlobalItem>();
 			mItem.addonSlotType = 1;
-			mItem.addonDmg = 2.6f;
-			mItem.addonHeat = 0.5f;
-			mItem.addonSpeed = -0.3f;
+			mItem.addonDmg = Common.Configs.MConfigItems.Instance.damageIceBeam;
+			mItem.addonHeat = Common.Configs.MConfigItems.Instance.overheatIceBeam;
+			mItem.addonSpeed = Common.Configs.MConfigItems.Instance.speedIceBeam;
```
Comment (Armipotent) [inline Content/Items/Addons/V3/StardustBeamAddon.cs]: Should Stardust Beam pull from Ice Beam values?

File: Content/Items/Addons/V3/VortexBeamAddon.cs [hunk 4]
```diff
@@ -36,9 +36,9 @@ public override void SetDefaults()
 			Item.createTile = ModContent.TileType<Content.Tiles.ItemTile.Beam.VortexBeamTile>();
 			MGlobalItem mItem = Item.GetGlobalItem<MGlobalItem>();
 			mItem.addonSlotType = 3;
-			mItem.addonDmg = 1.5f;
-			mItem.addonHeat = 1f;
-			mItem.addonSpeed = 0.25f;
+			mItem.addonDmg = Common.Configs.MConfigItems.Instance.damageIceBeam;
+			mItem.addonHeat = Common.Configs.MConfigItems.Instance.overheatIceBeam;
+			mItem.addonSpeed = Common.Configs.MConfigItems.Instance.speedIceBeam;
```
Comment (Armipotent) [inline Content/Items/Addons/V3/VortexBeamAddon.cs]: Again, why are all of the pillar beams pulling from Ice Beam?

**Review by Armipotent — CHANGES_REQUESTED** [review]
Some of the Tier-3 Beams are pulling from Ice Beam's config values, and that doesn't seem intentional. Was there a mistake perchance?

**Review by Antinoe — COMMENTED** [review]
No, none of this was intentional. I simply forgot to rename them correctly.

**Pull request merged**